requirements/docs.txt

```diff
@@ -139,41 +139,41 @@ snowballstemmer==2.2.0
     # via sphinx
 sortedcontainers==2.4.0
     # via hypothesis
-soupsieve==2.5
+soupsieve==2.6
     # via beautifulsoup4
-sphinx==7.3.7
+sphinx==8.0.2
```

Comment:
Please check that the docs are ok. Including figures, theme-specifics, etc.

Reply:
The build locally looks fine, nothing sticks out.